pkgs/os-specific/linux/firmware/facetimehd-firmware/default.nix

```diff
@@ -2,16 +2,16 @@
 
 let
 
-  version = "1.43_4";
+  version = "1.43_5";
 
 
   # Updated according to https://github.com/patjak/bcwc_pcie/pull/81/files
   # and https://github.com/patjak/bcwc_pcie/blob/5a7083bd98b38ef3bd223f7ee531d58f4fb0fe7c/firmware/Makefile#L3-L9
   # and https://github.com/patjak/bcwc_pcie/blob/5a7083bd98b38ef3bd223f7ee531d58f4fb0fe7c/firmware/extract-firmware.sh
 
   # From the Makefile:
-  dmgUrl = "https://support.apple.com/downloads/DL1877/en_US/osxupd10.11.5.dmg";
-  dmgRange = "205261917-208085450"; # the whole download is 1.3GB, this cuts it down to 2MB
+  dmgUrl = "https://updates.cdn-apple.com/2019/cert/041-88431-20191011-e7ee7d98-2878-4cd9-bc0a-d98b3a1e24b1/OSXUpd10.11.5.dmg";
+  dmgRange = "204909802-207733123"; # the whole download is 1.3GB, this cuts it down to 2MB
```

Comment:
How do you define this range ?

Reply:
@thpham https://github.com/patjak/facetimehd-firmware/issues/1#issuecomment-549585472

Reply:
See here: https://github.com/patjak/facetimehd-firmware/issues/1#issuecomment-549585472

Reply:
Basically the idea is that you find xz streams and offsets of these inside dmg file and then get files inside these streams and find a matching one. For end range I did bissection based on if I get required file from stream or not